.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+### Description
+What does this PR do?
+
+### TODO
+- [ ] Make sure there's a changelog. Brand new features have a differently
```

Comment:
This template will apply globally. I think (for the most part) we may want to have different guidance for, say, a docs PR. I _think_ we can address this in the future by adding a `.github/docs/pull_request_template.md` and so on -- the sky's the limit!

Reply:
I can add another template for docs PRs